app/wyze_bridge.py

```diff
@@ -434,25 +435,29 @@ def get_webrtc(self):
         signal.pause()
 
     def get_cameras(self) -> dict[str, dict]:
-        r: dict[str,dict] = {}
+        r: dict[str, dict] = {}
         for cam in self.cameras:
-            img = f"{self.img_path}{cam.name_uri}.jpg"
-            d:dict = {}
-            d['nickname'] = cam.nickname
-            d['ip'] = cam.ip
-            d['mac'] = cam.mac
-            d['product_model'] = cam.product_model
-            d['model_name'] = cam.model_name
-            d['firmware_ver'] = cam.firmware_ver
-            d['thumbnail'] = cam.thumbnail
-            d['hls_url'] = self.wb_hls_url + cam.name_uri + "/"
+            d: dict = {}
+            d["nickname"] = cam.nickname
+            d["ip"] = cam.ip
+            d["mac"] = cam.mac
+            d["product_model"] = cam.product_model
+            d["model_name"] = cam.model_name
+            d["firmware_ver"] = cam.firmware_ver
+            d["thumbnail"] = cam.thumbnail
+            d["hls_url"] = self.wb_hls_url + cam.name_uri + "/"
             if env_bool("LLHLS"):
-                d['hls_url'] = d['hls_url'].replace("http:", "https:")
-            d['rtmp_url'] = self.wb_rtmp_url + cam.name_uri
-            d['rtsp_url'] = self.wb_rtsp_url + cam.name_uri
-            d['name_uri'] = cam.name_uri
-            d['connected'] = self.streams[cam.nickname]["connected"].is_set() if "connected" in self.streams[cam.nickname] else False
-            d['img'] = img if os.path.exists(img) else None
+                d["hls_url"] = d["hls_url"].replace("http:", "https:")
+            d["rtmp_url"] = self.wb_rtmp_url + cam.name_uri
+            d["rtsp_url"] = self.wb_rtsp_url + cam.name_uri
+            d["name_uri"] = cam.name_uri
+            d["connected"] = (
+                self.streams[cam.nickname]["connected"].is_set()
+                if "connected" in self.streams[cam.nickname]
+                else False
+            )
+            if img := os.path.exists(f"{self.img_path}{cam.name_uri}.jpg"):
+                d["img"] = f"/local/{cam.name_uri}.jpg" if self.hass else img
```

Comment:
If it's HASS shouldn't {self.img_path} have already been set correctly upstream, it should be safe to prefix with img_path in either mode.